File: dolphin/docs/code/test_score_player.js
//run:
//java -classpath .:../../bin Run test_score_player.js JavaScript

importPackage(Packages.api.model);
importPackage(Packages.api.midi);

var Do=new Note(60);
var Re=new Note(62);
var Mi=new Note(64);

var part=new Part();
part.add(Do);
part.add(Re);
part.add(Mi);

var score=new Score();
score.add(part);

var player=new BasicScorePlayer();
player.play(score);

File: dolphin/docs/code/TestScorePlayer.java
import api.model.*;
import api.midi.*;

class TestScorePlayer {
  public static void main(String[] args) {
    final Note noteDo=new Note(60);
    final Note noteRe=new Note(62);
    final Note noteMi=new Note(64);

    final Part part=new Part();
    part.add(noteDo);
    part.add(noteRe);
    part.add(noteMi);

    final Score score=new Score();
    score.add(part);

    final ScorePlayer player=new BasicScorePlayer();
    player.play(score);
  }
}
